bluetooth/src/jsMain/kotlin/Value.kt

```diff
@@ -17,7 +17,13 @@
 
 package com.splendo.kaluga.bluetooth
 
+/**
+ * The JavaScript value of a bluetooth attribute
+ */
 actual typealias Value = ByteArray
```

Comment:
Just curious, since this is `actual` to some `expect` maybe it should get kdoc from the expect if it's missing in actual. I see a lot of duplications of a simple kdoc lines

Reply:
Unfortunately that is not how it works